src/features/student/views/StudentTraceView/StudentTraceView.vue

```diff
@@ -55,6 +55,10 @@ const breadcrumbLinks = computed(() => [
   { text: t('student.navigation.tabs.tools.items.traces'), to: studentToolsTracesRoute },
   { text: traceDetailed.value?.title || '' }
 ])
+
+watch(showUpdateModal, () => {
+  refetch()
+})
```

Comment:
Tu veux ce refetch à chaque ouverture et fermeture de modale ? Pas juste à l'ouverture ou la fermeture ?

```suggestion
watch(showUpdateModal, (newVal) => {
  if(!newval) { // à la fermeture // if(newval) pour à l'ouverture
    refetch()
  }
})
```